caluma/extensions/conftest.py

```diff
@@ -25,20 +25,32 @@ def user():
 
 
 @pytest.fixture
-def document_review_case(db, user):
-    case = start_case(
-        workflow=Workflow.objects.get(pk="document-review"),
-        form=Form.objects.get(pk="circulation-form"),
-        user=user,
+def create_document_review_case(db, user, form_question_factory):
+    form_question = form_question_factory(
+        question__type=Question.TYPE_TEXT, form__is_published=True
     )
 
-    Answer.objects.create(
-        question_id="circulation-decision",
-        document=case.document,
-        value="circulation-decision-approved",
-    )
+    def func():
+        case = start_case(
+            workflow=Workflow.objects.get(pk="document-review"),
+            form=form_question.form,
+            user=user,
+        )
+
+        Answer.objects.create(
+            question=form_question.question,
+            document=case.document,
+            value="foo",
+        )
+
+        return case
 
-    return case
+    return func
+
+
+@pytest.fixture
+def document_review_case(create_document_review_case):
+    return create_document_review_case()
```

Comment:
I think you could get rid of this indirection: The `create_document_review_case`'s inner function takes no params, so you could just do it instead of returning a callable. This would then also remove the need for the second fixture.

Reply:
I need this fixture thrice in one single test, that's why I went with the indirection.